core/domain/exp_domain.py

```diff
@@ -1777,59 +1777,61 @@ def create_default_state(
             feconf.DEFAULT_CONTENT_IDS_TO_AUDIO_TRANSLATIONS)
 
     @classmethod
-    def convert_html_fields_in_state(cls, state_dict, conversion_fn):
+    def convert_html_fields_in_state(cls, state_dict, conversion_fn, exp_id):
```

Comment:
Let's use the default argument here and make it more generic. So something like `convert_html_fields_in_state(cls, state_dict, conversion_fn, additional_data={})` then when you need some additional data you can call `exp_domain.convert_html_fields_in_state(state_dict, conversion_fn, additional_data={'exp_id': exploaration_id})` and when you don'!t you just call `exp_domain.convert_html_fields_in_state(state_dict, conversion_fn)`

Reply:
Done!